src/components/App.js

```diff
@@ -31,8 +31,11 @@ const App = () => {
     setAppState({ ...appState, CurrentPage: CurrentPage });
   };
   const SetSection = (Section, Content) => {
-    setAppState({ Resume: { ...appState.Resume, [Section]: Content } });
-   };
+    setAppState({
+      Resume: { ...appState.Resume, [Section]: Content },
+      CurrentPage: Section,
```

Comment:
No trailing commas please. What was change though?

Reply:
Ah, you added a `CurrentPage` to what's in `Section`?

Reply:
Yes, actually on changing the inputs, it's not setting the current page, so that's why nothing is shown. I have added CurrentPage: Section; to SetAppState, so now after changing the inputs, CurrentPage is again set to the current page.

Reply:
@MitAbhay One question: Does it work?

Reply:
Yes, Of course, you can check that.